### PR TITLE
Add rubygem-openscap_parser package

### PR DESCRIPTION
**Pull request by xprazak2**

### CODE REVIEW

**Comment by xprazak2** [comment]
Nokogiri 1.6.1 is in epel

File: packages/plugins/rubygem-openscap_parser/rubygem-openscap_parser.spec [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+%install
+mkdir -p %{buildroot}%{gem_dir}
+cp -a .%{gem_dir}/* \
+        %{buildroot}%{gem_dir}/
+
+%files
+%dir %{gem_instdir}
+%exclude %{gem_instdir}/.gitignore
+%exclude %{gem_instdir}/.travis.yml
+%{gem_instdir}/CODE_OF_CONDUCT.md
```
Comment (ekohl) [inline packages/plugins/rubygem-openscap_parser/rubygem-openscap_parser.spec]:
```suggestion
%doc %{gem_instdir}/CODE_OF_CONDUCT.md
```

File: comps/comps-foreman-plugins-rhel7.xml [hunk 2]
```diff
@@ -152,6 +152,7 @@
       <!--
         Do not edit this section manually and use ./comps_doc.sh script
       -->
+      <packagereq type="default">rubygem-openscap_parser</packagereq>
```
Comment (ekohl) [inline comps/comps-foreman-plugins-rhel7.xml]: Looks like the script places this in a weird location. Can you move it above the comment in a sorted location?

**Comment by ehelms** [comment]
@xprazak2 Can you update this when you get a chance and we'll get it in

**Comment by xprazak2** [comment]
It turns out we need additional fixes in this gem, I will reopen once the new version is released. Thanks!

**Pull request closed without merging**